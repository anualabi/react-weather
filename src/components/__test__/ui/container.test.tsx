import { render, screen } from "@testing-library/react";

import Container, { ContainerProps } from "@/components/ui/container";

const mockProps = {
  children: <div>children</div>,
  className: "",
};

const customRender = (props: ContainerProps) => {
  return render(<Container {...props} />);
};

describe("Container", () => {
  it("should render (snapshot)", () => {
    const { asFragment } = customRender({ ...mockProps });
    expect(asFragment()).toMatchSnapshot();
  });

  it("should render children", () => {
    customRender({ ...mockProps });
    expect(screen.getByText("children")).toBeInTheDocument();
  });
});
