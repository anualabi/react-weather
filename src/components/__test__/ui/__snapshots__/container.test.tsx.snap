// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render (snapshot) 1`] = `
<DocumentFragment>
  <div
    class="max-w-[1440px] w-full mx-auto px-5"
  >
    <div>
      children
    </div>
  </div>
</DocumentFragment>
`;
